### PR TITLE
feat(indexer): Make DB configurable in config.toml

### PR DESCRIPTION
**Pull request by roninjin10**
Make db configurable in the default toml file. Use the same env variables the old indexer used for easy migration

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6992** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6992" target="_blank">Graphite</a> 
    * **PR #6994** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6994" target="_blank">Graphite</a> 
      * **PR #6995** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6995" target="_blank">Graphite</a> 
        * **PR #6996** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6996" target="_blank">Graphite</a>  👈
          * **PR #7001** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7001" target="_blank">Graphite</a> 
            * **PR #7002** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7002" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6996?utm_source=stack-comment).

File: indexer/indexer.toml [hunk 1]
```diff
@@ -10,11 +10,15 @@ l1-rpc = "${INDEXER_RPC_URL_L1}"
 l2-rpc = "${INDEXER_RPC_URL_L2}"
 
 [db]
-host = "postgres"
+host = "$INDEXER_DB_HOST"
+# this port may be problematic once we depoly
```
Comment (hamdiallam) [inline indexer/indexer.toml:14]: If i had to guess, using the `postgresql://` scheme probably defaults to the 5432 ports

Quick google search confirms the default: https://www.prisma.io/dataguide/postgresql/short-guides/connection-uris -- we can remove this comment

**Review by hamdiallam — APPROVED** [review]

File: indexer/indexer.toml (hunk 1 above)
Comment (roninjin10) [reply indexer/indexer.toml:14]: I think I've actually had an issue in the past if I remember correctly when deploying gateway-backend. Gateway-backend didn't work the first time I deployed it and I had to remove the port

Comment (roninjin10) [reply indexer/indexer.toml:14]: Definitely a chance I'm remembering wrong though it was a long time ago

**Comment by roninjin10** [comment]
Converted back to draft because I forgot to make the `preset` chain id configurable

**Comment by roninjin10** [comment]
this already landed in another pr

**Pull request closed without merging**